java/java-impl/src/inspectionDescriptions/EnhancedSwitchBackwardMigration.html

```diff
@@ -26,6 +26,11 @@
 }
 </code></pre>
 <!-- tooltip end -->
+<p>
+  This inspection can help to downgrade for backward compatibility with Java 13 or earlier.
+  <i>Enhanced</i> <code>switch</code> appears in Java 14.
```

Comment:
I'd skip the second sentence, it's kind of clear from the first, no?

Reply:
OK so I have inverted the two sentences and I have replaced `Java 13 or earlier` by `earlier Java versions` to remove the redundancy.

I have also replaced `appears` with `appeared` because it's the past.

Is it OK?

Reply:
Yes, thanks!